spec/system/jobseekers_can_manage_their_saved_jobs_spec.rb

```diff
@@ -10,12 +10,64 @@
   context "when logged in" do
     before do
       login_as(jobseeker, scope: :jobseeker)
-      visit jobseekers_saved_jobs_path
     end
 
-    it "shows their saved jobs" do
-      # TODO: Implement me properly when saved jobs are implemented
-      expect(page).to have_content(I18n.t("jobseekers.saved_jobs.index.page_title"))
+    context "and there are saved jobs" do
+      let(:school) { create(:school) }
+      let(:vacancy_1) { create(:vacancy) }
+      let(:vacancy_2) { create(:vacancy) }
+      let(:expired_vacancy) { create(:vacancy, :expired) }
+
+      before do
+        vacancy_1.organisation_vacancies.create(organisation: school)
+        vacancy_2.organisation_vacancies.create(organisation: school)
+        expired_vacancy.organisation_vacancies.create(organisation: school)
+
+        SavedJob.find_or_create_by(jobseeker_id: jobseeker.id, vacancy_id: vacancy_1.id)
```

Comment:
to be consistent with what you just did on line 22 you could do:

`jobseeker.saved_jobs.create(vacancy: vacancy_1)`

do not think you need to do `find_or_create_by` in a spec